st2api/st2api/controllers/resource.py

```diff
@@ -312,16 +311,6 @@ def _get_by_name_or_id(self, name_or_id, exclude_fields=None):
 
         return resource_db
 
-    def _get_from_model_kwargs_for_request(self, **kwargs):
```

Comment:
IIRC, there was a reason I added this method in the past (something with the messy cross-test state being preserved because it first was a class variable and not instance variable), but if tests and everything pass I'm fine with removing it.